modules/AlphaWalletFoundation/AlphaWalletFoundation/Transfer/TransactionConfigurator.swift

```diff
@@ -176,7 +176,7 @@ public class TransactionConfigurator {
 
     public func shouldUseEstimatedGasPrice(_ estimatedGasPrice: BigUInt) -> Bool {
         //Gas price may be specified in the transaction object, and it will be if we are trying to speedup or cancel a transaction. The replacement transaction will be automatically assigned a slightly higher gas price. We don't want to override that with what we fetch back from gas price estimate if the estimate is lower
-        if let specifiedGasPrice = transaction.gasPrice, specifiedGasPrice > estimatedGasPrice {
+        if let specifiedGasPrice = transaction.gasPrice, specifiedGasPrice.max > estimatedGasPrice {
```

Comment:
This will be changed later when we support sending with ERC-1559 right? Would you help to add a `//TODO: support ERC-1559` and similar in other places you can think of?


Never mind if you already got it handled